backend/src/zac/accounts/admin.py

```diff
@@ -191,3 +193,15 @@ def formfield_for_dbfield(self, db_field, request, **kwargs):
             )
 
         return super().formfield_for_dbfield(db_field, request, **kwargs)
+
+
+# Unregister old GroupAdmin
+admin.site.unregister(Group)
+
+# Register new GroupAdmin
+@admin.register(Group)
+class GroupAdmin(_GroupAdmin):
+    form = GroupAdminForm
+
+    def get_queryset(self, request):
+        return super().get_queryset(request).prefetch_related("user_set")
```

Comment:
```suggestion
        return super().get_queryset(request).prefetch_related("user_set")
        
    def get_users_display(self, obj):
        return self.display_related_as_list_of_links(obj, "user_set")

    get_users_display.short_description = _("users")
```

There is only the name of the groups in the list display, I think it would be nice to also add the list of the users there. `RelatedLinksMixin` can be used for it.

Reply:
What if you get like 20 users in a group? Wouldn't that be super messy?

Reply:
Yeah you are right. It'd better the other way around but user's list page is already very busy. So please skip this coment